R/app-server.R

```diff
@@ -385,6 +385,9 @@ app_server <- function(input, output, session) {
 
         ## Upload only the files that have been given
         if (!is.null(indiv())) {
+          if (!is_name_valid(files$indiv$name)) {
+            stop("Please check that file names only contains: letters, numbers, spaces, underscores, hyphens, periods, plus signs, and parentheses.") # nolint
+          }
```

Comment:
This is a lot of repetition, with one per file to be uploaded. Doing this for the documentation filenames is going to be a hassle, as well, due to the unknown number of files (a user can upload 1+ assay documentation files). What do you think about adding this check to `save_to_synapse()`?

Reply:
Yeah that seems good!